demo/portal/src/app/services/constants.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
We may need to figure out where to put this as it's duplicated across systems. 
Marking as TODO for myself.